core/Date.php

```diff
@@ -605,7 +615,38 @@ public function subYear($n)
      * Returns a localized date string using the given template.
      * The template should contain tags that will be replaced with localized date strings.
      *
-     * Allowed tags include:
+     * @param string $template eg. `"MMM y"`
+     * @return string eg. `"Aug 2009"`
+     */
+    public function getLocalized($template)
+    {
+        $template = $this->replaceLegacyPlaceholders($template);
```

Comment:
Does it make sense to do this only if `$template` starts with `%`? I'm not sure how often `getLocalized` is called... If it is not often called it won't be needed I reckon. Thinking more re performance